Formula/sourcery.rb

```diff
@@ -13,11 +13,10 @@ class Sourcery < Formula
   depends_on :xcode => ["8.0", :build]
 
   def install
-    ENV.delete("CC")
-    ENV["SDKROOT"] = MacOS.sdk_path
-    system "swift", "build", "-c", "release"
-    bin.install ".build/release/sourcery"
-    lib.install Dir[".build/release/*.dylib"]
```

Comment:
Install all private files into `libexec` (which Homebrew will not try to put elsewhere) and then symlink what you need into `bin`, `lib`, etc.